lib/terms_of_use/exceptions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module TermsOfUse
+  module Exceptions
+    class CommonNameNotFoundError < StandardError; end
```

Comment:
This should be `CommonNameMissingError` or something along those lines (since 'not found' suggests we queried and didn't find it)